scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -14,10 +14,11 @@
 # don't use postgres
 os.environ["ROX_POSTGRES_DATASTORE"] = "false"
 
-versions=get_latest_release_versions(4)
+# versions=get_latest_release_versions(4)
+chart_versions=["72.1.0", "71.2.0", "70.0.0"]
 
 gkecluster=GKECluster("qa-e2e-test")
```

Comment:
while you are here can you also:
```suggestion
gkecluster=GKECluster("compat-test")
```
(I picked something short because long names can have issues)